{
  "loc.friendlyName": "Azure CLI",
  "loc.helpMarkDown": "[追加情報](http://go.microsoft.com/fwlink/?LinkID=827160)",
  "loc.description": "Azure サブスクリプションに対して Azure CLI コマンドを使用してシェル スクリプトまたはバッチ スクリプトを実行します",
  "loc.instanceNameFormat": "Azure CLI $(scriptPath)",
  "loc.releaseNotes": "バージョン 1.0 の新機能:\n- Python ベースの新しい Azure CLI 2.0 をサポート\n- クロスプラットフォーム エージェント対応 (Linux、macOS、Windows)\n- Azure CLI 1.0 (node.js ベース) に対応するためのタスク バージョン 0.0 への切り替え\n- 制限事項:\n - Azure クラシック サブスクリプションはサポートされていません。Azure CLI 2.0 では Azure Resource Manager (ARM) サブスクリプションのみがサポートされています。",
  "loc.group.displayName.advanced": "詳細設定",
  "loc.input.label.connectedServiceNameARM": "Azure サブスクリプション",
  "loc.input.help.connectedServiceNameARM": "展開用の Azure Resource Manager サブスクリプションを選択します",
  "loc.input.label.scriptLocation": "スクリプトの場所",
  "loc.input.help.scriptLocation": "スクリプトの種類: ファイル パスまたはインライン スクリプト",
  "loc.input.label.scriptPath": "スクリプト パス",
  "loc.input.help.scriptPath": "スクリプトの完全修飾パス、または既定の作業ディレクトリに対する相対パス",
  "loc.input.label.inlineScript": "インライン スクリプト",
  "loc.input.help.inlineScript": "インラインのスクリプトをここに記述できます。バッチ ファイルについては、すべての azure コマンドの前にプレフィックス \"call\" を使用します。また、引数を使用して定義済み変数やカスタム変数をこのスクリプトに渡すこともできます\n\n シェルの例: az --version || az account show \n\n バッチの例: call  az --version || call az account show",
  "loc.input.label.args": "引数",
  "loc.input.help.args": "スクリプトに渡される引数",
  "loc.input.label.cwd": "作業ディレクトリ",
  "loc.input.help.cwd": "スクリプトを実行する現行の作業ディレクトリ。空白はリポジトリ (ビルド) または成果物 (リリース) のルートであり、$(System.DefaultWorkingDirectory) となります",
  "loc.input.label.failOnStandardError": "標準エラーで失敗",
  "loc.input.help.failOnStandardError": "これが true の場合、StandardError ストリームにエラーが書き込まれると、このタスクは失敗します。チェック ボックスを選択解除して標準エラーを無視し、終了コードから状態を確認します",
  "loc.messages.ScriptReturnCode": "スクリプトが次のリターン コードで終了しました: %d",
  "loc.messages.ScriptFailed": "スクリプトがエラーで失敗しました: %s",
  "loc.messages.UnsupportedEndpointScheme": "サポートされていないエンドポイント認証スキーム: AzureRM に対するサービス プリンシパル",
  "loc.messages.AzureSDKNotFound": "Azure CLI 2.x はこのマシンにインストールされていません。",
  "loc.messages.FailedToLogout": "ログアウト中に次のエラーが発生しました: %s"
}